test/test_function_schema.py

```diff
@@ -27,11 +27,14 @@ def test_backward_compatible_args(self):
         self.assertFalse(old_schema.is_backward_compatible_with(new_schema))
 
     def test_backward_compatible_kwargs(self):
-        old_schema = parse_schema('any(Tensor self, *, Tensor out) -> Tensor')
```

Comment:
The current test organization is difficult to read and maintain.

I would expect that a single schema is created and each of the properties are tested sequentially with a comment starting the block that tests them. For example:

```
# Tests output narrowing
  <new schema where output is wider fails>
  <new schema where output is narrow passes>

# Tests arg count
  < new schema with fewer args fails>
```

Reply:
Agreed. Updated tests with new organization and comments.